apps/design-land/src/app/form/form.component.html

```diff
@@ -0,0 +1,12 @@
+<form [formGroup]="form" #ngForm="ngForm">
+    <daff-form-field>
+      <input daff-input type="text"
+        placeholder="Daff Input Demo"
+        [formControl]="form.controls['inputTest1']"
+        [formSubmitted]="ngForm.submitted">
+      <daff-error-message *ngIf="form.controls['inputTest1'].errors && (form.controls['inputTest1'].touched || ngForm.submitted)">This is a required field >:(</daff-error-message>
+      <div class="form__error-placeholder" *ngIf="!(form.controls['inputTest1'].errors && (form.controls['inputTest1'].touched || ngForm.submitted))"></div>
```

Comment:
I thought about this a bit more after on conversation. I think I shouldn't add a margin to the form-field because what if the user doesn't have an error message? Then they just have this extra margin there for nothing. What do you think?

Reply:
@lderrickable I think it's safe to assume they will. In the event they don't, it most likely serves a niche purpose. I brought a similar issue like this up in Angular Material long ago:

https://github.com/angular/material2/issues/8644 which led to https://github.com/angular/material2/issues/7145#issuecomment-330282110 which I think still has yet to be implemented.

Reply:
@lderrickable In the docs, you put ``daff-form-field` is a wrapper for form control elements that is used to organize and display error messages along with their associated form control elements.`

Given this requirement, I think it does make sense for form-field to have margin/padding? `daff-form-field` would not be used if there is no error message attached to a form control?

Reply:
@xelaint I don't think that's true either. I think a form field would be useful even if we don't have error messages.

Reply:
done